pkg/http/github.go

```diff
@@ -52,12 +52,8 @@ func CheckCLIVersion(versionCheck models.VersionCheck, silent bool, json bool, d
 	utils.LogDebug("Checking for latest version of the CLI")
 	tag, err := getLatestVersion()
 	if err != nil {
-		if debug && !json {
-			utils.LogError(err)
-		}
-		if !silent && !json {
-			utils.LogError(errors.New("Unable to check for CLI updates"))
-		}
+		utils.LogError(errors.New("Unable to check for CLI updates"))
+		utils.LogDebugError(err)
```

Comment:
I think is this a breaking change. We will have to notify all our customers to now use the `--silent` flag.

Reply:
This isn't a breaking change. This command already uses the `--silent` flag.

Reply:
Sorry what I mean is that users who are using the `--json` flag are expecting no logging on CLI update failures. Now they will need to provide `--silent` flag to get the same effect.

Reply:
Ah gotcha. It's true that this now prints when running with `--json` and `-debug`, however it was already standard practice to always print debug messages, even when using `--json`. Thus this was the only place that didn't follow this. Therefore the breakiness is a little fuzzy to me because anyone calling `--json --debug` could already expect to see non-json debug messages.

Reply:
Okay that makes sense. I think it is fine then.